dockerfiles/pytorch-dev/install_rocm_from_tarballs.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# -----------------------------------------------------------------------------
+# install_rocm_from_tarballs.sh
+#
+# Download and install ROCm tarballs for specified AMDGPU targets from
+# TheRock's nightly GitHub releases.
+#
+# Usage:
+#   ./install_rocm_from_tarballs.sh "gfx942 gfx1100"
+#
+# Environment Variables (optional):
+#   RELEASE_TAG, ROCM_VERSION_DATE, ROCM_VERSION_PREFIX, INSTALL_PREFIX,
+#   OUTPUT_ARTIFACTS_DIR
+#
+# Requirements:
+#   curl (auto-installed if missing), bash
+# -----------------------------------------------------------------------------
+
+# Ensure curl is installed
+if ! command -v curl >/dev/null 2>&1; then
+  echo "[INFO] curl not found. Installing..."
+  if command -v apt-get >/dev/null 2>&1; then
+    apt-get update
+    apt-get install -y curl
+  else
+    echo "[ERROR] curl installation not supported on this OS. Please install it manually."
+    exit 1
+  fi
+fi
+
+# Configuration
+RELEASE_TAG="${RELEASE_TAG:-nightly-release}"
+ROCM_VERSION_DATE="${ROCM_VERSION_DATE:-$(date -d '3 days ago' +'%Y%m%d')}"
+ROCM_VERSION_PREFIX="6.4.0rc"
```

Comment:
I could image that it would be nice for users to set `ROCM_VERSION` instead (we can still use `ROCM_VERSION_{DATE,PREFIX}` internally) or to pass the version as an argument to the script.

Reply:
now it ready the version from version.json file

Reply:
Himm, make sense.

Reply:
Updated based on option 1 and tested